docs/api.md

```diff
@@ -444,7 +444,7 @@ Emitted when a [response] is received.
 
 #### page.$(selector)
 - `selector` <[string]> A [selector] to query page for
-- returns: <[Promise]<[ElementHandle]>>
+- returns: <[Promise]<[ElementHandle] | `null`>>
```

Comment:
here and everywhere: let's use question mark to define nullable types.
E.g.:

```
returns: <[Promise]<?[ElementHandle]>>
```

This would align with jsdoc notation we currently use.

Reply:
Thanks for pointing that out. I have wrongly taken JSDocs notation to be the same as typescript's, in which optional type is denoted with `?` (`undefined`) and `null` value has to be explicitly stated. Apparently, in JSDocs `?` means nullable types.

I have updated the PR to question marks as required.